### ISSUE TITLE
Calibration

### ISSUE THREAD

**Issue by lukemorse** [LuukEbenau/SacaDev.Muse#6]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
It would be great to have a calibration feature similar to what Muse has done with their own app.

**Comment by LuukEbenau** [comment]
I agree with that this is a quite manatory feature, which most people would need. However i'm not quite sure how we can implement this feature properly.

One potentional implementation:
1. Add a Calibrate(seconds) method to the Muse class.
2. Add a signaladdress for Eeg_calibrated (or per frequency range, like SignalAddress.Delta_Calibrated, enz.)  
This allows subscribing to the calibrated signals.

I am also not quite sure yet how to implement the calibration method. Anyone some suggestions for algorithms which can be of use?

**Comment by lukemorse** [comment]
I found this in the forum:
http://forum.choosemuse.com/t/favorite-algorithms-for-eeg-features-analysis/487/4


Calculating the mean and standard deviation is easy enough, but he says he
feeds it into a leaky-integrator.  I looked it up and it seems a bit
outside of my skill set.

**Comment by LuukEbenau** [comment]
It's also not really inside my own toolset, but perhaps I (or someone else) can make an initial suboptimal version of an calibration method.
